src/index.js

```diff
@@ -1,4 +1,4 @@
 import React from 'react';
 import createAll from './components/createAll';
 
-export const { Provider, Connector, provide, connect } = createAll(React);
+export const { Provider, Connector, provide, connect, connectDeprecated } = createAll(React);
```

Comment:
@gaearon should it still export Connector and connectDeprecated rather than removing it completely ?